packages/interface/src/components/interface-skeleton/index.js

```diff
@@ -95,6 +95,7 @@ function InterfaceSkeleton(
 	const [ secondarySidebarResizeListener, secondarySidebarSize ] =
 		useResizeObserver();
 	const isMobileViewport = useViewportMatch( 'medium', '<' );
+	const isLargeViewport = useViewportMatch( 'medium' );
```

Comment:
Suggestion (non-blocking): Haven't tested this. Can we check if it's a large viewport by using negation instead of adding another listener?

```
isLargeViewport = ! isMobileViewport
```

Reply:
Yea, good call. Adjusted it.